### DISCUSSION TITLE
如何設定 HTTPS port

### DISCUSSION THREAD

**Discussion by M-02** [1715173329/UnblockNeteaseMusic#420] [Q&A] [answered]
运行unblockneteasemusic-linux-x64  -p 16163 指定端口为16163的时候，不会启动https代理，只有一个http代理，不指定端口有http和https两个代理，指定端口运行的话网易云客户端就会显示无法连接网络
![image](https://user-images.githubusercontent.com/34387208/147538105-9e7859e6-fded-472d-a7a2-c510e234acb2.png)

**Comment by 1715173329** [comment, marked as answer]
端口格式为 http:https
e.g. -p 16163:16164

**Reply by M-02** [reply]
好的，谢谢大佬

**Comment by pan93412** [comment]
要啟用 HTTPS 功能的話，需要這樣寫：`-p HTTP:HTTPS`

假如你希望 Port 80 是 HTTP，Port 443 是 HTTPS，那就需要寫成 `-p 80:443`。解決了幫我點一下 Mark as answer，有問題歡迎再提出～

**Reply by M-02** [reply]
谢谢大佬
